subscriber/src/daily_operations.rs

```diff
@@ -0,0 +1,227 @@
+use core::borrow::Borrow;
+
+use auto_farm::common::{address_to_id_mapper::AddressId, unique_payments::UniquePayments};
+use multiversx_sc_modules::ongoing_operation::{CONTINUE_OP, STOP_OP};
+use subscription_fee::subtract_payments::{MyVeryOwnScResult, ProxyTrait as _};
+
+use crate::{
+    base_functions::SubscriberContract,
+    service::{PaymentType, SubscriptionType},
+};
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+pub type Epoch = u64;
+
+pub const DAILY_EPOCHS: Epoch = 1;
+pub const WEEKLY_EPOCHS: Epoch = 7;
+pub const MONTHLY_EPOCHS: Epoch = 30;
+
+pub const FIRST_INDEX: usize = 1;
+pub const GAS_TO_SAVE_PROGRESS: u64 = 100_000;
+
+#[derive(TypeAbi, TopEncode, TopDecode, Default)]
+pub struct OperationProgress {
+    pub service_index: usize,
+    pub current_index: usize,
+    pub additional_data_index: usize,
+}
+
+#[multiversx_sc::module]
+pub trait DailyOperationsModule:
+    crate::service::ServiceModule
+    + crate::user_tokens::UserTokensModule
+    + crate::common_storage::CommonStorageModule
+    + energy_query::EnergyQueryModule
+    + multiversx_sc_modules::ongoing_operation::OngoingOperationModule
+{
+    fn perform_service<SC: SubscriberContract<SubSc = Self>>(
```

Comment:
This is like the whole implementation of xExchange premium accounts auto claim boosted feature. like the subscriber. Can you make a specific folder for this ? like the name should represent this. 

Also in the folder subscriber - we should have only a template of how should a subscriber look like.

Reply:
It would be better to see this under another folder.

Reply:
Users won't have to interact with this at all. I've provided an example in the metabonding-subscriber folder.

Reply:
Is this the base code for any subscriber implementation ?